plugins/entity-validation/dev/index.tsx

```diff
@@ -16,9 +16,18 @@
 import React from 'react';
 import { createDevApp } from '@backstage/dev-utils';
 import { entityValidationPlugin, EntityValidationPage } from '../src/plugin';
+import { CatalogApi, catalogApiRef } from '@backstage/plugin-catalog-react';
 
 createDevApp()
   .registerPlugin(entityValidationPlugin)
+  .registerApi({
+    api: catalogApiRef,
+    deps: {},
+    factory: () =>
+      ({
+        getEntities: () => ({}),
+      } as CatalogApi),
+  })
```

Comment:
Is there a reason to add this? I haven't tried the dev mode to be honest, do we need to register the api to make it work?

Reply:
You do. This just made it easier to iterate quickly on loading it up and inspecting some of the functionality. It's kind of handy, and could be extended in the future if desired.

Reply:
Good to know! So let's keep it then